go/pkg/hydra/store/sqlc-queries-analysis.md

```
@@ -0,0 +1,625 @@
+        store.WorkflowStatusFailed,
+        now,
+        store.WorkflowStatusSleeping,
+        now,
+    ).
+    Where("workflow_name IN ?", workflowNames). // Optional filter
+    Order("created_at ASC").
+    Offset(offset).
+    Limit(limit)
+```
+**SQL Equivalent**:
+```sql
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)